src/scripts/rubocop-check.sh

```diff
@@ -2,13 +2,23 @@
 
 mkdir -p "$PARAM_OUT_PATH"
 
+RUBOCOP_VERSION=$(bundle exec rubocop -v)
+RUBOCOP_VERSION_MAJOR=$(echo "$RUBOCOP_VERSION" | cut -d. -f1)
+RUBOCOP_VERSION_MINOR=$(echo "$RUBOCOP_VERSION" | cut -d. -f2)
+
 if [ "$PARAM_PARALLEL" -eq 1 ]; then
   bundle exec rubocop "$PARAM_CHECK_PATH" \
   --format "$PARAM_FORMAT" \
   --out $"$PARAM_OUT_PATH"/check-results.xml
```

Comment:
If I understand well, parallel is default from 1.32, so this won't be parallel in older versions. It should enforce it to have it always.

Reply:
That’s correct. Thank you for pointing that out, and I’ve made the necessary adjustments.